### DISCUSSION TITLE
Use midi keyboard?

### DISCUSSION THREAD

**Discussion by Stugist** [FundamentalFrequency/.github#87] [Q&A]
I have an AKAI MPK Mini MKII, would it be possible to use it instead of the custom kit?

**Comment by iamdey** [comment]
I didn't find any way to do it but I'm interested.

**Comment by Stugist** [comment]
hmmm....i guess the real deciding factor is if there's any drivers that work for it on rPi. even a generic that reads most of the inputs would get me where i'm going.

like, i have an 8" touch screen in a 3d printed enclosure with hinges on, 3 rPIs, and the keyboard. guess it's time to start looking for drivers for the keyboard.
